src/runtime_src/core/tools/xbmgmt2/SubCmdDump.cpp

```diff
@@ -34,22 +37,50 @@ namespace po = boost::program_options;
 // ------ L O C A L   F U N C T I O N S ---------------------------------------
 
 static void
-flash_dump(std::ofstream& /*fOutput*/)
+flash_dump(const std::string& output, const std::shared_ptr<xrt_core::device>& _dev)

```

Comment:
Pass in the device and then the output, for consistency purposes.

Reply:
will change